### ISSUE TITLE
last major CS2 update broke csgove

### ISSUE THREAD

**Issue by maxpain** [akiver/csgo-voice-extractor#3]
```bash
root@44b46bf89688:/# LD_LIBRARY_PATH=/csgove /csgove/csgove -output /tmp/voices /12262741_12332313_2402101331-de_ancient.dem
Processing demo /12262741_12332313_2402101331-de_ancient.dem
Failed to parse demo: /12262741_12332313_2402101331-de_ancient.dem

runtime error: index out of range [7] with length 7
stacktrace:
goroutine 19 [running]:
runtime/debug.Stack()
	runtime/debug/stack.go:24 +0x5e
github.com/markus-wa/demoinfocs-golang/v4/pkg/demoinfocs.NewParserWithConfig.func1({0x7aeaa0?, 0xc0009e2030})
	github.com/markus-wa/demoinfocs-golang/v4@v4.0.0/pkg/demoinfocs/parser.go:388 +0x2f
github.com/markus-wa/godispatch.(*Dispatcher).dispatchWithRecover.func1()
	github.com/markus-wa/godispatch@v1.4.1/dispatch.go:197 +0x3b
panic({0x7aeaa0?, 0xc0009e2030?})
	runtime/panic.go:914 +0x21f
github.com/markus-wa/godispatch.callConsumerCode.func1()
	github.com/markus-wa/godispatch@v1.4.1/dispatch.go:122 +0x54
panic({0x809f00?, 0xc00054d968?})
	runtime/panic.go:914 +0x21f
github.com/markus-wa/demoinfocs-golang/v4/pkg/demoinfocs/sendtables2.glob..func17(0xc000ac41e0?, 0xc0005764e0)
	github.com/markus-wa/demoinfocs-golang/v4@v4.0.0/pkg/demoinfocs/sendtables2/field_path.go:55 +0x5f
github.com/markus-wa/demoinfocs-golang/v4/pkg/demoinfocs/sendtables2.readFieldPaths(0x0?, 0xc00016f608)
	github.com/markus-wa/demoinfocs-golang/v4@v4.0.0/pkg/demoinfocs/sendtables2/field_path.go:326 +0x17c
github.com/markus-wa/demoinfocs-golang/v4/pkg/demoinfocs/sendtables2.(*Entity).readFields(0xc000148930, 0xc0002f1470?, 0xc00016f608)
	github.com/markus-wa/demoinfocs-golang/v4@v4.0.0/pkg/demoinfocs/sendtables2/entity.go:416 +0x45
github.com/markus-wa/demoinfocs-golang/v4/pkg/demoinfocs/sendtables2.(*Parser).OnPacketEntities(0xc00014c140, 0xc0006a7520)
	github.com/markus-wa/demoinfocs-golang/v4@v4.0.0/pkg/demoinfocs/sendtables2/entity.go:527 +0x618
reflect.Value.call({0x75f500?, 0xc00012c438?, 0xc00016fe28?}, {0x86390c, 0x4}, {0xc00016fef8, 0x1, 0x846280?})
	reflect/value.go:596 +0xce7
reflect.Value.Call({0x75f500?, 0xc00012c438?, 0x58?}, {0xc00016fef8?, 0x1700000000000001?, 0x3?})
	reflect/value.go:380 +0xb9
github.com/markus-wa/godispatch.callConsumerCode({0x75f500?, 0xc00012c438?, 0x44d1a5?}, {0xc00016fef8?, 0x419026?, 0x43e08e?})
	github.com/markus-wa/godispatch@v1.4.1/dispatch.go:125 +0x3f
github.com/markus-wa/godispatch.(*Dispatcher).Dispatch(0xc0001222a0, {0x8450a0, 0xc0006a7520})
	github.com/markus-wa/godispatch@v1.4.1/dispatch.go:111 +0x1d4
github.com/markus-wa/godispatch.(*Dispatcher).dispatchWithRecover(0xc0001222b8?, {0x8450a0?, 0xc0006a7520?})
	github.com/markus-wa/godispatch@v1.4.1/dispatch.go:202 +0x4f
github.com/markus-wa/godispatch.(*Dispatcher).dispatchQueue(0xc0001222a0, 0x0?)
	github.com/markus-wa/godispatch@v1.4.1/dispatch.go:177 +0xd0
created by github.com/markus-wa/godispatch.(*Dispatcher).AddQueues in goroutine 1
	github.com/markus-wa/godispatch@v1.4.1/dispatch.go:162 +0x18a
No voice data found in demo /12262741_12332313_2402101331-de_ancient.dem

End processing demo /12262741_12332313_2402101331-de_ancient.dem
```

**Comment by maxpain** [comment]
@akiver, I think we need to update `demoinfocs-golang`

**Comment by maxpain** [comment]
https://github.com/DandrewsDev/CS2VoiceData/issues/4

**Comment by akiver** [comment]
Fixed as of v2.1.0, thank you!
